themes/jaeger-docs/assets/sass/pages.sass

```
@@ -52,12 +52,13 @@
       position: fixed
       bottom: 0%
       top: 9rem
+      left: 3rem 
 
       .nav
         position: absolute
         z-index: 10
-        width: inherit
-        max-width: 11rem
+        width: 180px
+        max-width: 20rem
```

I tried this instead.
```suggestion
        min-width: 15rem
        max-width: 25rem
```
My version works better as Perf Tuning no longer gets wrapped. However neither setting works at 150% zoom or higher, the sidebar starts overlapping the main text (but that's probably the issue with how the main area is defined).